common/src/platform/CMakeLists.txt

```diff
@@ -1,11 +1,14 @@
 cmake_minimum_required(VERSION 3.0)
 project(lg_common_platform LANGUAGES C)
 
+add_library(lg_common_platform INTERFACE)
+target_link_libraries(lg_common_platform INTERFACE lg_common_platform_code)
+
 if (UNIX)
   add_subdirectory("linux")
 elseif(WIN32)
   add_subdirectory("windows")
+  if (ENABLE_BACKTRACE)
+    target_link_libraries(lg_common_platform INTERFACE dbghelp)
```

Comment:
Why not just add this to `platform/windows/CMakeLists.txt`?

Reply:
Did not see that file. Fixed.